src/tespy/networks/networks.py

```diff
@@ -2511,6 +2511,134 @@ def postprocessing(self):
         msg = 'Postprocessing complete.'
         logging.info(msg)
 
+    def exergy_analysis(self, pamb=101300, Tamb=298.15, bus=None, E_F=None,
+                        E_L=None):
+        r"""Perform exergy analysis.
+
+        - Get values for physical exergy of connections.
+        - Call exergy balances of components.
+          The components cycle\_closer, source, sink and splitter are being
+          skipped because it does not make sense to perform an exergy analysis
+          with these components.
+        - Calculate exergy destruction and exergetic efficiency of components.
+          If there is a componenten in the network for which there is no
+          exergy balance implemented in TESPy, 0 is assigned to E_D, E_F and
+          E_P of the componenten. The exergetic efficiency epsilon gets the
+          value -111.
```

Comment:
I am not so sure about this. Can we use np.nan instead?

Reply:
Yes, of course.
I did not think of that approach. It convinces me.